scripts/init-env.sh

```
@@ -109,12 +109,16 @@ GO_BACKEND_URL=http://localhost:$BACKEND_PORT
-SENTRY_DSN=${SENTRY_DSN:-}
-EOF
+NEXT_PUBLIC_POSTHOG_HOST=${NEXT_PUBLIC_POSTHOG_HOST:-}
+NEXT_PUBLIC_POSTHOG_UI_HOST=${NEXT_PUBLIC_POSTHOG_UI_HOST:-}
+POSTHOG_PROXY_PATH=${POSTHOG_PROXY_PATH:-}
+POSTHOG_PROXY_TARGET=${POSTHOG_PROXY_TARGET:-}
+POSTHOG_PROXY_ASSETS_HOST=${POSTHOG_PROXY_ASSETS_HOST:-}
+
+    SENTRY_ORG=${SENTRY_ORG:-sorami}
+    SENTRY_PROJECT=${SENTRY_PROJECT:-lang-portal-frontend}
+    NEXT_PUBLIC_SENTRY_DSN=${NEXT_PUBLIC_SENTRY_DSN:-}
+    EOF
```

Shellcheck (0.11.0)

[error] Remove indentation before end token (or use <<- and indent with tabs).

(SC1039)